parser/src/approximator.rs

```diff
@@ -163,7 +168,7 @@ mod tests {
         sync::mpsc::{self},
     };
 
-    use crate::prelude::*;
+    use crate::{ast::MulType, prelude::*};
```

Comment:
add multype to pub(crate) prelude?
```suggestion
    use crate::prelude::*;
```